src/rules/ssl.ts

```diff
@@ -84,11 +84,11 @@ export const ssl: Rule = async (
       violation: ViolationType.NONE,
       description: `Valid SSL settings.`
     }
-  } catch (error) {
```

Comment:
Kun je het error-type niet gebruiken?

Reply:
Ik zou niet weten wat het error type zou kunnen wezen?
Zonder `: any` gaat de code kapot wanneer je data (like de message) probeerd te gebruiken. 
Alle andere plekken waar we error data proberen te lezen gebruiken we ook de `: any` methode.